app/assets/javascripts/admin/templates/users-list-show.hbs

```diff
@@ -22,62 +22,87 @@
         {{/if}}
         <th>&nbsp;</th>
-      </tr>
-
-      {{#each model as |user|}}
-        <tr class="user {{user.selected}} {{unless user.active 'not-activated'}}">
-          {{#if showApproval}}
+      </thead>
+      <tbody>
+        {{#each model as |user|}}
+          <tr class="user {{user.selected}} {{unless user.active 'not-activated'}}">
+            {{#if showApproval}}
             <td>
               {{#if user.can_approve}}
                 {{input type="checkbox" checked=user.selected}}
               {{/if}}
             </td>
-          {{/if}}
-          <td><a href="{{unbound user.path}}" data-user-card="{{unbound user.username}}">{{avatar user imageSize="small"}}</a></td>
-          <td>{{#link-to 'adminUser' user}}{{unbound user.username}}{{/link-to}}</td>
-          <td class='email'>{{unbound user.email}}</td>
-          <td>{{{unbound user.last_emailed_age}}}</td>
-          <td>{{{unbound user.last_seen_age}}}</td>
-          <td>{{number user.topics_entered}}</td>
-          <td>{{number user.posts_read_count}}</td>
-          <td>{{{unbound user.time_read}}}</td>
+            {{/if}}
+            <td>
+              <a href="{{unbound user.path}}" data-user-card="{{unbound user.username}}">
+                {{avatar user imageSize="small"}}
+              </a>
+            </td>
+            <td>
+              {{#link-to 'adminUser' user}}{{unbound user.username}}{{/link-to}}
+            </td>
+            <td class='email'>
+              {{unbound user.email}}
+            </td>
+            <td>
+              {{{unbound user.last_emailed_age}}}
+            </td>
+            <td>
+              {{{unbound user.last_seen_age}}}
+            </td>
+            <td>
+              {{number user.topics_entered}}
+            </td>
+            <td>
+              {{number user.posts_read_count}}
+            </td>
+            <td>
+              {{{unbound user.time_read}}}
+            </td>
 
-          <td>{{{unbound user.created_at_age}}}</td>
+            <td>
+              {{{unbound user.created_at_age}}}
+            </td>
 
-          {{#if showApproval}}
-          <td>
-            {{#if user.approved}}
-              {{i18n 'yes_value'}}
-            {{else}}
-              {{i18n 'no_value'}}
+            {{#if showApproval}}
+              <td>
+                {{#if user.approved}}
+                  {{i18n 'yes_value'}}
+                {{else}}
+                  {{i18n 'no_value'}}
+                {{/if}}
+              </td>
             {{/if}}
-          </td>
-          {{/if}}
-          <td>
-            {{#if user.admin}}<i class="fa fa-shield" title="{{i18n 'admin.title'}}"></i>{{/if}}
-            {{#if user.moderator}}<i class="fa fa-shield" title="{{i18n 'admin.moderator'}}"></i>{{/if}}
-          </td>
-        </tr>
-      {{/each}}
-
+            <td>
+              {{#if user.admin}}
+                <i class="fa fa-shield" title="{{i18n 'admin.title'}}"></i>
```

Comment:
@mikechristopher look like i cannot. The problem is not with `{{fa-icon "shield"}}` but with using handlerbars helpers in form elment. 
Did you see `{{fa-icon "shield"}}` used in form tag?